css/blocked-seller.css

```diff
@@ -3,6 +3,8 @@
 }
 
 .blocked-seller .seller_info strong a,
+.blocked-seller a,
+.blocked-seller a strong,
```

Comment:
These rules are not specific enough and end up causing all anchors within a blocked seller's item to be affected:
<img width="1088" alt="Screen Shot 2021-09-21 at 9 14 12 PM" src="https://user-images.githubusercontent.com/12960237/134283285-aeb0a8c8-24d3-4866-abd3-dfc48a40840a.png">

It should look like the following:
<img width="1084" alt="Screen Shot 2021-09-21 at 9 15 53 PM" src="https://user-images.githubusercontent.com/12960237/134283311-1b78d101-39aa-4cd0-b1f6-83bde7c9cb18.png">